core/databases/db_completion_tasks.py

```diff
@@ -32,9 +36,30 @@ def db_get_completion_tasks_by_page(page: int, per_page: int = defaults.ITEMS_PE
     return results
 
 
+def db_set_incomplete_completion_task_executing(uuid: str):
```

Comment:
`incomplete` is unnecessary here
```suggestion
def db_set_completion_task_executing(uuid: str):
```